src/dlgProfilePreferences.cpp

```diff
@@ -631,6 +640,13 @@ void dlgProfilePreferences::setDisplayFont()
         pHost->mDisplayFont = font;
         if (mudlet::self()->mConsoleMap.contains(pHost)) {
             mudlet::self()->mConsoleMap[pHost]->changeColors();
+
```

Comment:
These lines mean that the main console gets updated as soon as the control is adjusted in this dialogue and thus doing the same thing in the `slot_save_and_exit()` is redundant` - do we want the change to happen here or when the dialogue is saved (there is no obvious cancel to avoid the save although the close button in the title bar _may_ achieve a cancel without a save)?